HEAD.js

```diff
@@ -0,0 +1,36 @@
+const request = require('request');
+const {execSync} = require('child_process');
+const {CloudBuildClient} = require('@google-cloud/cloudbuild');
+
+const client = new CloudBuildClient();
+
+/**
+ * @returns {Promise<string>}
+ */
+const getDeployedHead = () => {
```

Comment:
Random thought, rather than use the word "Head" throughout the code, I wonder if it might be quicker to grok if we referred to this as the "version"? I realize it's not the same as the package.json version, but IMO it makes some of the code easier to read at a glance, e.g. `if (deployedVersion === currentVersion)`